musicbrainz/api_test.go

```diff
@@ -0,0 +1,110 @@
+// This file is part of the go-meta library.
+//
+// Copyright (C) 2017 JAAK MUSIC LTD
+//
+// This program is free software: you can redistribute it and/or modify
+// it under the terms of the GNU Lesser General Public License as published by
+// the Free Software Foundation, either version 3 of the License, or
+// (at your option) any later version.
+//
+// This program is distributed in the hope that it will be useful,
+// but WITHOUT ANY WARRANTY; without even the implied warranty of
+// MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+// GNU General Public License for more details.
+//
+// You should have received a copy of the GNU Lesser General Public License
+// along with this program.  If not, see <http://www.gnu.org/licenses/>.
+//
+// If you have any questions please contact yo@jaak.io
+
+package musicbrainz
+
+import (
+	"bytes"
+	"database/sql"
+	"encoding/json"
+	"fmt"
+	"net/http"
+	"net/http/httptest"
+	"testing"
+
+	"github.com/meta-network/go-meta"
+	"github.com/neelance/graphql-go"
+)
+
+// TestArtistAPI tests querying an artist index via the GraphQL API.
+func TestArtistAPI(t *testing.T) {
+	// create a test index of artists
+	x, err := newTestIndex()
+	if err != nil {
```

Comment:
Is it ok to reference test code from other `_test.go` files?

Reply:
Yes, as long as it's in the same package.